runbotics/runbotics-orchestrator-ui/src/src-app/Actions/types.ts

```diff
@@ -19,6 +19,10 @@ export enum CloudPath {
     SITE = 'site',
 }
 
+export enum Patterns {
+    VARIABLE_INPUT = '^[a-zA-Z0-9_]*$',
+}
```

Comment:
do we really need enum here? I don't see what could be included here. `Pattern` is a broad term, so better narrow it to the scope of actions like `OutputVariablePattern` as a const.

fyi next time name enum in the singular form